paypal/paypal-messages-android
# ci: functional tests

## Description

Adds various instrumentation tests to increase coverage and ensure reliability.

## Testing instructions

- Review the tests and corresponding code to make sure the code is well tested and no tests are missing. 
- Run the tests to make sure they pass.
